server/rel/overlays/bin/web

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd -P -- "$(dirname -- "$0")"
+TUIST_WEB=true TUIST_WORKER=false exec ./tuist start
```

Comment:
I decided to keep the `server` overlay to run both, and then introduce `web` and `worker` which are more granular.